cg/cli/deliver/base.py

```diff
@@ -137,6 +146,13 @@ def concatenate(context: click.Context, ticket: str, dry_run: bool):
     help="Deliver and rsync the files for ALL cases connected to a ticket",
     required=True,
 )
+@click.option(
```

Comment:
since it is repeated - should we make it a constant like `@FORCE_ALL`

Reply:
That makes sense 👍